assets/grafana/deployment.yaml

```diff
@@ -43,6 +44,12 @@ spec:
           requests:
             cpu: 4m
             memory: 64Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          readOnlyRootFilesystem: true
```

Comment:
I think this change will introduce a bug https://github.com/prometheus-operator/kube-prometheus/issues/1623 - there is a fix in progress upstream